demo/src/pick_place_task.cpp

```diff
@@ -217,6 +225,7 @@ void PickPlaceTask::init() {
   ---- *               Allow Collision (hand object)   *
 		 ***************************************************/
 		{
+			// Modify the planning scene (does not alter the robot's position) to permit picking up the object.
```

Comment:
```suggestion
			// Modify the planning scene (does not alter the robot's position) to allow collision between hand and object. Otherwise, the robot would not be able to move after picking the object.
```